src/components/MenuMob.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styled from 'styled-components'
+import Layout from 'atomic-layout'
+import { Link } from "react-router-dom"
+import { menu } from './Menu'
+
+const StyleMenuMob = styled.ul`
+    position: absolute;
+    top: 10vh;
+    right: 0;
+    padding: 5px;
+    color: black;
+    transition: transform 0.7s;
+
+    transform: translateX(${({ isOpen }) => isOpen ? 0 : '100%'});
+
+    @media (min-width: ${Layout.breakpoints.md.minWidth}) {
```

Comment:
This is a good example how can you use atomic layout breakpoints. Look at it if you have problems with my previous comment.